src/relax/transform/remove_redundant_assignments.cc

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*!
+ * \file src/relax/transform/remove_redundant_assignments.cc
+ * \brief This pass removes redundant assignment statements. These stmts are result of other pass
+ * like hint_on_device processed by RealizeVDevice may leave them. The subsequent pass like
+ * fuse_ops fail to fuse in this case.
```

Comment:
is it the same as https://github.com/apache/tvm/blob/main/src/relax/transform/canonicalize_bindings.cc?

Reply:
Yes. Both are same.